tests/Filesystem/FilesystemTest.php

```diff
@@ -267,6 +267,17 @@ public function testIsWritable()
         $this->assertTrue($files->isWritable($this->tempDir.'/foo.txt'));
     }
 
+    public function testIsReadable()
```

Comment:
This test is not strong enough since it past before when the implementation was wrong. Make sure the test files arn't writable.